### PR TITLE
Added social media URL to articles for twitter integration

### PR DESCRIPTION
**Pull request by adamboeglin**

### CODE REVIEW

File: docs/example-scenario/update_social_images.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```
Comment (MikeWasson) [inline docs/example-scenario/update_social_images.py]: Can we move this file to the /build folder?

File: docs/example-scenario/ai/movie-recommendations.md [hunk 2]
```diff
@@ -1,131 +1,132 @@
...
-[ref-arch]: /azure/architecture/reference-architectures/ai/real-time-recommendation
-[regions]: https://azure.microsoft.com/en-us/global-infrastructure/services/?products=virtual-machines&regions=all
-[resiliency]: /azure/architecture/resiliency/
-[sec-docs]: /azure/security/
-[setup]: https://github.com/Microsoft/Recommenders/blob/master/SETUP.md%60
-[sla]: https://azure.microsoft.com/en-us/support/legal/sla/virtual-machines/v1_8/
-[sla-aks]: https://azure.microsoft.com/en-us/support/legal/sla/kubernetes-service/v1_0/
-[storage-security]: /azure/storage/common/storage-service-encryption
-[vm-size]: /azure/virtual-machines/virtual-machines-linux-change-vm-size
+---
```
Comment (MikeWasson) [inline docs/example-scenario/ai/movie-recommendations.md]: This file shows a large diff. Maybe need to rebase?

Comment (adamboeglin) [reply docs/example-scenario/ai/movie-recommendations.md]: The line endings were in windows format (CRLF) and were changed to Unix style encoding after being parsed by the script.  It now matches the rest of the docs, so this is preferred anyway.

File: docs/example-scenario/update_social_images.py (hunk 1 above)
Comment (adamboeglin) [reply docs/example-scenario/update_social_images.py]: Done.

File: docs/example-scenario/ai/movie-recommendations.md (hunk 2 above)
Comment (adamboeglin) [reply docs/example-scenario/ai/movie-recommendations.md]: The file was changed from windows style line endings (CRLF) to Unix style.  No text was actually changed.

**Comment by adamboeglin** [comment]
Comments resolved.

**Pull request merged**